config.yaml

```diff
@@ -140,6 +140,7 @@ params:
     nw: 400
     c: "-c"
     hm: 3
+    gtf: "/hpf/largeprojects/ccm_dccforge/dccdipg/Common/annotation/exons/hg19_UCSC_exons_canonical_placeholder.bed"
```

Comment:
Is this the same bed used for the SV report (line 100)? If so can we just use the same file?

Reply:
It is the same bed, but since qualimap requires a bed of 6 columns and the sv report bed contains only 4, I added two columns of "." as placeholders. Thus named as "_placeholder.bed".